community/contributing/updating_the_class_reference.rst

```diff
@@ -1,151 +1,176 @@
 .. _doc_updating_the_class_reference:
 
-Contributing to the class reference
-===================================
+Contributing to the class reference with Git
+============================================
```

Comment:
This page is cross-referenced in several places as the starting page for contributing to the docs - but now it's specifically about the Git workflow.

I would suggest making sure that all references to this page are now done only for Git workflow purposes (and I would rename the file, reference and change the title further). General references should go to `doc_documentation_guidelines` (which likely needs a rename too - I'd rename all pages and their titles to be up to date with their current content).

Reply:
Fixing the references is done, they were all about editing the classes' XML or the git workflow specifically, so I linked to those two pages instead of the general docs writing guides.